config/checker_severity_map.json

```diff
@@ -118,10 +121,12 @@
   "cert-msc32-c":                                               "MEDIUM",
   "cert-msc50-cpp":                                             "LOW",
   "cert-msc51-cpp":                                             "MEDIUM",
+  "cert-mem57-cpp":                                             "MEDIUM",
   "cert-oop11-cpp":                                             "MEDIUM",
   "cert-oop54-cpp":                                             "MEDIUM",
   "cert-oop57-cpp":                                             "HIGH",
-  "cert-oop58-cpp":                                             "LOW",
+  "cert-oop58-cpp":                                             "MEDIUM",
+  "cert-pos44-c":                                               "MEDIUM",
```

Comment:
The severity level of this checker at [here](https://wiki.sei.cmu.edu/confluence/display/c/POS44-C.+Do+not+use+signals+to+terminate+threads) is also `LOW`.

Reply:
See the comment above